lldb/source/Plugins/Language/ObjC/Cocoa.cpp

```diff
@@ -131,6 +131,20 @@ bool lldb_private::formatters::NSTimeZoneSummaryProvider(
       stream.Printf("%s", summary_stream.GetData());
       return true;
     }
+  } else if (class_name == "_NSSwiftTimeZone") {
+    llvm::ArrayRef<ConstString> identifier_path = {
+        ConstString("timeZone"),
+        ConstString("_timeZone"),
+        ConstString("some"),
+        ConstString("identifier"),
```

Comment:
This almost looks like some leftover debugging code?

Reply:
It's not debugging code. That's the variable path that contains the time zone's name (effectively `timeZone._timeZone.identifier`).

Reply:
Oh right `.some` is the enumerator for Optional!
"some identifier"  just sounded like a placeholder :-)